cypress/e2e/tests/migration/controls/businessservices/create.test.ts

```diff
@@ -91,7 +92,13 @@ describe(["@tier2"], "Business service validations", () => {
     });
 
     it("Business service success alert and unique constraint validation", function () {
-        const businessService = new BusinessServices(data.getCompanyName());
+        const stakeholder = new Stakeholders(data.getEmail(), data.getFullName());
```

Comment:
Is stakeholder mandatory for creating BS ? If yes then we need to clean up after the test